Frameworks/UIKit/UILongPressGestureRecognizer.mm

```diff
@@ -251,6 +251,26 @@ - (void)touchesCancelled:(NSSet*)touches withEvent:(UIEvent*)event {
     [self reset];
 }
 
+/**
+ @Status Interoperable
+*/
+- (CGPoint)locationInView:(UIView*)view {
+    if (_trackedTouches.size() > 0) {
+        CGPoint averagePoint = { 0, 0 };
+        for (size_t i = 0; i < _trackedTouches.size(); i++) {
```

Comment:
>size_t i = 0; i < _trackedTouches.size(); i++ [](start = 13, length = 45)

Should use range-based for;

for (TrackedTouch& trackedTouch : _trackedTouches) { ...